icons/package.svg

```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <line x1="16.5" y1="9.4" x2="7.5" y2="4.21" />
+  <line x1="16.5" y1="9.4" x2="7.8" y2="4.4" />
```

Comment:
It now doesn't extend far enough:
![image](https://user-images.githubusercontent.com/17746067/228304762-52413053-1330-49fd-b98f-7ddfaa65769c.png)